libs/pageserver_api/src/config.rs

```diff
@@ -102,6 +102,7 @@ pub struct ConfigToml {
     pub ingest_batch_size: u64,
     pub max_vectored_read_bytes: MaxVectoredReadBytes,
     pub image_compression: ImageCompressionAlgorithm,
+    pub timeline_offloading: bool,
```

Comment:
should it be an Option bool or have a default value so that old config can be decoded?

Reply:
ahh, sounds good